lib/apiservers/engine/backends/container.go

```diff
@@ -117,6 +119,38 @@ const (
 	DefaultEnvPath         = "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"
 )
 
+func NotFoundError(msg string) error {
+	return derr.NewRequestNotFoundError(fmt.Errorf("No such container: %s", msg))
+}
+
+func UnknownError(msg string) error {
```

Comment:
Why do we have both unknownerror and server error? They seem to serve much the same role